tox.ini

```diff
@@ -159,22 +164,24 @@ deps =
 whitelist_externals =
     {[base]whitelist_externals}
 commands =
-    bash {toxinidir}/.travis/runflake8.sh {envpython} --statistics {posargs} .
+    bash {toxinidir}/.travis/runflake8.sh --exclude=.venv,.tox --statistics {posargs} .
 
 [testenv:yamllint]
 deps = yamllint
+whitelist_externals =
```

Comment:
This env does not use bash, so the `whitelist_externals` does not seem to be needed here.

Reply:
it is needed because of the `command_pre` in `[testenv]` which applies to every section . . . so I suppose the right answer here is to remove the `command_pre`.  This means `setup_module_utils.sh` will not be done automatically by default for every test env.  I suppose I could add a `command_pre` for `setup_module_utils.sh` to every env, but that seems fragile and redundant, because in the case of network, it isn't needed for `pytest`.  So I guess the right answer is to add a hook in the scripts that might need to set up module_utils for testing, such as `.travis/runpytest.sh`, `.travis/runpylint.sh`, `.travis/runflake8.sh`, that is controlled by `.travis/config.sh` and allows the user to specify that module_utils needs to be setup for testing.

Reply:
done - see latest commit